definitions/npm/rxjs_v5.0.x/flow_v0.29.0-/rxjs_v5.0.x.js

```diff
@@ -247,35 +332,35 @@ declare class rxjs$Observable<+T> {
 
   subscribe(observer: rxjs$PartialObserver<T>): rxjs$Subscription;
   subscribe(
-    onNext: ?(value: T) => mixed,
-    onError: ?(error: any) => mixed,
-    onCompleted: ?() => mixed,
+    next: ?(value: T) => mixed,
+    error: ?(error: any) => mixed,
+    complete: ?() => mixed,
   ): rxjs$Subscription;
 
   static combineLatest<A, B>(
     a: rxjs$Observable<A>,
-    resultSelector: (a: A) => B,
+    resultSelector: ?(a: A) => B,
```

Comment:
Could you add tests to make sure that this still works as expected, with and without the `resultSelector` argument?